test/puppetlabs/puppetdb/cli/benchmark_test.clj

```diff
@@ -86,18 +105,18 @@
        (#'cli-export/-main "--outfile" export-out-file
                            "--host" (:host svc-utils/*base-url*)
                            "--port" (str (:port svc-utils/*base-url*)))))
-    (let [submitted (run-benchmark {}
-                                   "--config" "anything.ini"
-                                   "--numhosts" "2"
-                                   "--nummsgs" "3"
-                                   "--archive" export-out-file)]
-      (is (= 18 (count submitted))))))
+    (let [submitted (benchmark-nummsgs {}
+                                       "--config" "anything.ini"
+                                       "--numhosts" "2"
+                                       "--nummsgs" "3"
+                                       "--archive" export-out-file)]
+      (is (= (* 2 3 3) (count submitted))))))
```

Comment:
@mullr Worth one step further, i.e. let bind numhosts and nummsgs and use them here and in the args above?  Not sure.